sys-block/gpart/gpart-0.3~git.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A tool for recovering MBR partition tables in hard disks"
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are \
+on a PC type, MBR-partitioned hard disk in case the primary partition table \
+was damaged."
+HOMEPAGE="https://github.com/baruch/gpart"
+COPYRIGHT="1999-2017 Michail Brzitwa"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="23b651d1e9c6ae1454e5d3fe315d7d4e145661c0"
+SOURCE_URI="https://github.com/baruch/gpart/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="c492b7dc677e4c6af197f9598ba815a4e14a03578ee8a963430529767bc6c947"
+SOURCE_FILENAME="gpart-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="gpart-$srcGitRev"
+PATCHES="gpart-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	gpart = $portVersion
+	cmd:gpart = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc
+	cmd:make
+	"
+
+defineDebugInfoPackage gpart \
+	$binDir/gpart
```

Comment:
As mentioned earlier, this package is not built with `-g`, thus there won't be any debuginfo to extract

Reply:
```diff
-defineDebugInfoPackage gpart \
-	$binDir/gpart
-
```
Exact. We don't want a debuginfo package here.

If we comment out the call to `defineDebugInfoPackage gpart $binDir/gpart` and rebuild the recipe, then install the newly built package and run:
```bash
readelf /bin/gpart | grep debug
```
you will see no output. This means `-g` is not being passed to the compiler, as @alaviss has already noticed.